etc/picongpu/jureca-jsc/gpus.tpl

```diff
@@ -27,21 +27,26 @@
 # Sets batch job's name
 #SBATCH --job-name=!TBG_jobName
 #SBATCH --nodes=!TBG_nodes
+#SBATCH --exclusive
```

Comment:
Just to be sure: Did you test this profile on multiple nodes? I once made bad experience with `--exclusive` at ZIH. But I guess how this option is treated is system dependent.

Reply:
I tested it on 4 nodes and it works fine.

Reply:
Yes, I also tested it earlier and it was working fine, but I went through the documentation again and found [this](https://apps.fz-juelich.de/jsc/hps/jureca/known-issues.html#slurm-srun-options-exact-and-exclusive-change-default-pinning). Apparently as a side effect --exclusive pins to SMT cores instead of physical cores. Following the workaround listed, I will edit the script to add ``-m *:block``.